packages/webapp/src/components/Navigation/TopMenu/TopMenu.jsx

```diff
@@ -30,6 +30,7 @@ import {
 import Alert from '../../../containers/Navigation/Alert';
 import { useTranslation } from 'react-i18next';
 import clsx from 'clsx';
+import TextButton from '../../Form/Button/TextButton';
```

Comment:
Just wondering if we should continue using our custom TextButton? Or use one of MUI's buttons like IconButton or something moving forward.

Reply:
It felt awkward to me too that we were using TextButton for an image. IconButton seems good, thank you!